python/taichi/tools/np2ply.py

```diff
@@ -1,278 +1,278 @@
+                "ply\n", "format " + format + " 1.0\n",
+                "comment " + self.comment + "\n"
+            ])
+            f.write("element vertex " + str(self.num_vertices) + "\n")
+            for i in range(self.num_vertex_channels):
+                f.write("property " + self.vertex_data_type[i] + " " +
+                        self.vertex_channels[i] + "\n")
+            if (self.num_faces != 0):
+                f.write("element face " + str(self.num_faces) + "\n")
+                f.write("property list uchar int vertex_indices\n")
+                for i in range(self.num_face_channels):
+                    f.write("property " + self.face_data_type[i] + " " +
+                            self.face_channels[i] + "\n")
+            f.write("end_header\n")
+
+    def export(self, path):
+        self.sanity_check()
+        self.print_header(path, "binary_" + sys.byteorder + "_endian")
+        with open(path, "ab") as f:
+            for i in range(self.num_vertices):
+                for j in range(self.num_vertex_channels):
+                    f.write(self.vertex_data[j][i])
+            if self.face_type == "tri":
+                vert_per_face = np.uint8(3)
+            else:
+                vert_per_face = np.uint8(4)
+            for i in range(self.num_faces):
+                f.write(vert_per_face)
+                for j in range(vert_per_face):
+                    f.write(self.face_indices[i, j])
+                for j in range(self.num_face_channels):
+                    f.write(self.face_data[j][i])
+
+    def export_ascii(self, path):
+        self.sanity_check()
+        self.print_header(path, "ascii")
+        with open(path, "a") as f:
+            for i in range(self.num_vertices):
+                for j in range(self.num_vertex_channels):
+                    f.write(str(self.vertex_data[j][i]) + " ")
+                f.write("\n")
+            if self.face_type == "tri":
+                vert_per_face = 3
+            else:
+                vert_per_face = 4
+            for i in range(self.num_faces):
+                f.writelines([
+                    str(vert_per_face) + " ",
+                    " ".join(map(str, self.face_indices[i, :])), " "
+                ])
+                for j in range(self.num_face_channels):
+                    f.write(str(self.face_data[j][i]) + " ")
+                f.write("\n")
+
+    def export_frame_ascii(self, series_num: int, path: str):
+        # if path has ply ending
+        last_4_char = path[-4:]
+        if last_4_char == ".ply":
+            path = path[:-4]
+
+        real_path = path + "_" + "{0:0=6d}".format(series_num) + ".ply"
+        self.export_ascii(real_path)
+
+    def export_frame(self, series_num: int, path: str):
+        # if path has ply ending
+        last_4_char = path[-4:]
+        if last_4_char == ".ply":
+            path = path[:-4]
+
+        real_path = path + "_" + "{0:0=6d}".format(series_num) + ".ply"
```

Comment:
Oh, what makes the git have such a big changeset? Did we do something unexpected? ;O

Reply:
Our format server @taichi-gardener is replacing CRLF by LF introduced in #1706. Let's revert this part and let the code format to fail for now to pvc #1728.